html/changelogs/TheLion1675-Bugfixing.yml

```diff
@@ -0,0 +1,45 @@
+################################
+# Example Changelog File
+#
+# Note: This file, and files beginning with ".", and files that don't end in ".yml" will not be read. If you change this file, you will look really dumb.
+#
+# Your changelog will be merged with a master changelog. (New stuff added only, and only on the date entry for the day it was merged.)
+# When it is, any changes listed below will disappear.
+#
+# Valid Prefixes: 
+#   bugfix
+#   wip (For works in progress)
+#   tweak
+#   soundadd
+#   sounddel
+#   rscadd (general adding of nice things)
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#################################
+
+# Your name.  
+author: TheLion1675
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - bugfix: "Fixed a present bug where pulse rifles would not accept HV magazines."
+  - bugfix: "Fixed an issue in piping and power which caused issues in medical isolation."
+  - bugfix: "Fixed an issue in the CSCIO's office where two consoles weren't able to be interacted with."
+  - bugfix: "Increased SMO's office to fix a similar issue as above."
+  - bugfix: "Fixed the missing taser from the CSECO's locker."
+  - bugfix: "Fixed an accidental duplication of RIG spawns in the CSECO locker."
+  - rscdel: "Removed the flash from base locker spawns. Will be re-added when they are fixed. When? I don't know, ask Nanako."
```

Comment:
this is a bit passive aggressive for a changelog. Don't put in lines that would expose a team member to potential harassment